.github/workflows/land-blocking.yml

```diff
@@ -89,13 +116,35 @@ jobs:
                 ]
               }' > /tmp/payload
               curl -X POST -H 'Content-type: application/json' -d @/tmp/payload ${{ secrets.WEBHOOK_FLAKY_LAND_BLOCKING_CT }}
-              exit 1
+              exit 0
             else
               compat_prev_tag=$(tail -1 lbt_images_output.txt)
               echo "Using previous image tag $compat_prev_tag"
               echo "::set-env name=PREV_TAG::$compat_prev_tag"
             fi
           fi
+          echo "::set-env name=BUILD_PREV::0"
```

Comment:
This goes hand-in-hand with lines 99 and 109. It took me a while to line up the if/else to see what we are doing here. 

Can we add a comment here or beginning of this step to indicate this step will set the condition to build the images from the $BASE_GIT_REV iff they are not found and iff compat is enabled (ie, not being KS)?